spec/models/donut_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+require_relative "../../app/models/donut.rb"
```

Comment:
you should not have to require_relative as the `rails_helper` will do this for you :-)